### PR TITLE
Added Windows Support

### PR DESCRIPTION
**Pull request by abhi1693**
- Fixed getCpuinfo() for Windows

### CODE REVIEW

File: src/SI.php [hunk 1]
```diff
@@ -1,469 +1,524 @@
 <?php
```
Comment (abhi1693) [inline src/SI.php]: - Rearranged Code

**Comment by trntv** [comment]
Sorry for delay with answer. FOA, thanks for you PR.
I am rebuilding the whole library now, so will merge you code inside new version later.

**Comment by abhi1693** [comment]
I have also started work on the same library. Please take a look [here](https://github.com/abhi1693/yii2-system-info)

**Comment by trntv** [comment]
Yep, something like that
22 февр. 2015 г. 7:25 пользователь "Abhimanyu Saharan" <
notifications@github.com> написал:

**Pull request closed without merging**